File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.java
class Solution {
    public String maximumOddBinaryNumber(String s) 
    {
        int cnt_1 = 0;
        char res[] = s.toCharArray();
        for(int i=0;i<s.length();i++)
        {
            if(res[i] == '1'){
                cnt_1++;
            }
        }
        for(int i=0;i<cnt_1-1;i++){
            res[i] = '1';
        }
        for(int i=cnt_1-1;i<s.length()-1;i++){
            res[i] = '0';
        }
        res[s.length()-1] = '1';
        return String.valueOf(res);
    }
}

File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        cnt_1 = 0
        for i in s:
            if i == '1':
                cnt_1 += 1
        res = ''
        for i in range(cnt_1-1):
            res += '1'
        for i in range(len(s)-cnt_1):
            res += '0'
        res += '1'
        return res